### ISSUE TITLE
Issue with logging multiple confusion matricies

### ISSUE THREAD

**Issue by tomginsberg** [wandb/client#1030]
wandb, version 0.8.35
Python 3.7.7
Linux

### Description

I'm trying to log a confusion matrix after each epoch using 
`wandb.sklearn.plot_confusion_matrix(all_predictions, all_labels, class_labels,
                                            title=f'Confusion Matrix: Epoch {epoch}', normalize=True)`. 

My expected behaviour would be multiple confusion matrices getting logged each with the appropriate title, but the actual result (shown below), is that the original matrix gets overwritten and the individual values end up written on top of each other. 

![](https://i.imgur.com/HiJsIs4.png)

**Comment by cvphelps** [comment]
Hi @tomginsberg thanks for your issue. @lavanyashukla could you take a look

**Comment by lavanyashukla** [comment]
Hi @tomginsberg thanks for reporting this! Yup, we don't currently support confusion matrices across runs, but will have a fix out soon. We'll reach out to you when we ship it :)

**Comment by tomginsberg** [comment]
Thanks, guys. I'll probably implement a temporary solution myself and share it here for anyone with the same issue.

**Comment by tomginsberg** [comment]
Here's a good termporary fix 

```python
from sklearn.metrics import confusion_matrix
for epoch in range(num_epochs):
        ...
        cm = confusion_matrix(all_labels, all_predictions, normalize='pred')
        wandb.log({f'Confusion Matrix. Epoch {epoch}': wandb.plots.HeatMap(class_labels, 
                            class_labels, matrix_values=cm, show_text=True)})
```
Unfortunately plot labels don't appear (maybe there's a way to do this), but the most recent plot will be first on the media panel.

![](https://i.imgur.com/OkWdBof.png)

**Comment by ariG23498** [comment]
Hey @tomginsberg 
In the past year we've majorly reworked the CLI and UI for Weights & Biases. We're closing issues older than 6 months. Please comment to reopen.

**Comment by mmlynarik** [comment]
Hi, is this issue solved or not?

**Comment by JinsuHongg** [comment]
I got same issue. I try to log confusion matrix on every validation epoch. but the titles are not changed. all the titles are same, "Confusion Matrix Curve". I think argument "title" is not working?
def on_validation_epoch_end(self):
  # confusion matrix
              wandb.log(
                  {
                      f"conf_mat_epoch_{self.current_epoch}":
                      wandb.plot.confusion_matrix(
                          probs=None,
                          y_true=targets_list,
                          preds=preds_list,
                          class_names=self.class_names,
                          title=f"confusion_matrix_epoch_{self.current_epoch}"
                      )
                  })
